project/eScavenge/test_Game.py

```diff
@@ -114,8 +114,13 @@ def setUp(self):
         self.game.started = False
 
     def test_start_game(self):
+        self.assertEqual(Errors.GAME_STARTED, self.game.start(), "game already started")
+        self.assertTrue(self.game.started, "game in progress value not set")
+
+    def test_start_game_game_already_running(self):
         self.game.start()
```

Comment:
Not part of your change, but we shouldn’t call the method here, should just set the member.